src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -234,18 +239,21 @@ private static void gunzipInPlace(File file) {
         }
     }
 
-    private static void applySubsetSingleFileNcks(File file, GoGoDuckModule module) {
-        List<String> ncksSubsetParameters = module.getNcksSubsetParameters().getNcksParameters();
+    private void applySubsetSingleFileNcks(File file, GoGoDuckModule module, List<String> ncksSubsetParameters) {
         List<String> ncksExtraParameters = module.ncksExtraParameters();
 
         try {
             File tmpFile = File.createTempFile("tmp", ".nc");
 
             List<String> command = new ArrayList<String>();
-            command.add(GoGoDuckConfig.ncksPath);
+            command.add(goGoDuckConfig.getNcksPath());
             command.add("-a");
             command.add("-4");
             command.add("-O");
+            if (!module.isTimeUnlimited()) {
```

Comment:
Actually, perhaps this should just be moved into the code in module that returns nckssubsetparameters.  Kind of confusing creating working out the subset parameters in two places.